docs/How-To_Guides/Object_storage/s3-credentials.md

```diff
@@ -0,0 +1,53 @@
+# Working with S3-compatible credentials
+
+When you want to interact with object storage in {{extra.brand}} using
+tools that support an Amazon S3 compatible API (such as `s3cmd`,
+`rclone`, the `aws` CLI, or the Python `boto3` library), you need an
+S3-compatible access key and secret key.
+
+
+## Creating credentials
+
+You can create a set of S3-compatible credentials with the following
+command:
+
+```bash
+openstack ec2 credentials create
+```
+
+This will return an `Access` and `Secret` key that you can use to
+populate the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+environment variables (or whichever configuration options your
+application requires).
+
+!!! note
+    Your S3-compatible credentials are always scoped to your
+    {{extra.brand}} *region* and *project*. You cannot reuse an access
+    and secret key across multiple regions or projects.
+	
+	Also, your credentials are only “S3-compatible” in the sense that
+	they use the same *format* as AWS S3 does. They are never valid against
+	AWS S3 itself.
+
+
+## Listing credentials
+
+You can list any previously-created credentials with:
+
```

Comment:
should we mention somewhere as Note, that Secret is shown only once during credentials creation?

Reply:
No, because that's not true. :) `openstack ec2 credentials list` always lists the existing access and secret keys defined in a tenant.